sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyClientTest.java

```diff
@@ -76,6 +76,16 @@ public void createRsaKey(HttpClient httpClient, KeyServiceVersion serviceVersion
         createRsaKeyRunner((expected) -> assertKeyEquals(expected, client.createRsaKey(expected)));
     }
 
+    /**
+     * Tests that an Ec key is created.
+     */
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("getTestParameters")
+    public void createEcKey(HttpClient httpClient, KeyServiceVersion serviceVersion) {
+        getKeyClient(httpClient, serviceVersion);
+        createEcKeyRunner((expected) -> assertNotNull(client.createEcKey(expected)));
```

Comment:
Same comment as above :)

```suggestion
        createEcKeyRunner((expected) -> StepVerifier.create(client.createEcKey(expected))
            .assertNext(response -> assertKeyEquals(expected, response))
            .verifyComplete());
```